src/Compilers/VisualBasic/Portable/Binding/Binder.vb

```diff
@@ -327,6 +327,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        ''' <summary>
+        ''' Additional members associated with the binding context
+        ''' Currently, this field is only used for multiple field symbols initialized by an AsNew field initializer, e.g. "Dim x, y As New Integer"
+        ''' </summary>
+        Public Overridable ReadOnly Property AdditionalContainingMembers As ImmutableArray(Of Symbol)
```

Comment:
>AdditionalContainingMembers [](start = 45, length = 27)

It feels like this method should also be overridden in 
- Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator.ParametersAndLocalsBinder
- Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator.EENamedTypeBinder